data_steward/constants/validation/email_notification.py

```diff
@@ -1,5 +1,5 @@
-MANDRILL_API_KEY = 'MANDRILL_API_KEY'
-UNSET_MANDRILL_API_KEY_MSG = f"Mandrill API key not set in environment variable {MANDRILL_API_KEY}"
+MANDRILL_TOKEN = 'mandrill_token'
```

Comment:
Imo lets rename this to something that indicates its the name of a secret.  Maybe something like `MANDRILL_TOKEN_SECRET_ID` or something.  Not

Reply:
@dcarbone How about now?